src/cascade/dismod/db/metadata.py

```diff
@@ -534,7 +534,14 @@ class Var(Base):
     covariate_id = Column(Integer(), nullable=False)
 
 
-_TYPE_MAP = {str: String, int: Integer, float: Float, np.dtype("int64"): Integer, np.dtype("float64"): Float}
+_TYPE_MAP = {
```

Comment:
I think this map is my fault. I'm realizing I need to make the name more specific. It's a map from pandas types to sqlalchemy types, I think.